module/VuFind/src/VuFindTest/Feature/CacheManagementTrait.php

```diff
@@ -62,14 +62,26 @@ protected function getCacheClearPermissionConfig(): array
     }
 
     /**
-     * Call the API to clear object cache
+     * Call the API to clear object cache.
      *
-     * @return voi
+     * @return void
      */
     protected function clearObjectCache(): void
+    {
+        $this->clearCache('object');
+    }
+
+    /**
+     * Call the API to clear cache.
+     *
+     * @param string $name Cache name
+     *
+     * @return void
+     */
+    protected function clearCache(string $name): void
     {
         $http = new HttpService();
-        $client = $http->createClient($this->getVuFindUrl('/api/v1/admin/cache?id=object'), Request::METHOD_DELETE);
+        $client = $http->createClient($this->getVuFindUrl('/api/v1/admin/cache?id=' . $name), Request::METHOD_DELETE);
```

Comment:
I realize there is almost no chance of this being abused or problematic, but for formality, I think it might be best to:

```suggestion
        $client = $http->createClient($this->getVuFindUrl('/api/v1/admin/cache?id=' . urlencode($name)), Request::METHOD_DELETE);
```